.envrc

```diff
@@ -12,3 +12,4 @@ fi
 if [ -n $HAB_ORIGIN ]; then
   HAB_ORIGIN_KEYS="${HAB_ORIGIN_KEYS},${HAB_ORIGIN}"
 fi
+HAB_BLDR_CHANNEL=unstable
```

Comment:
Minor pedantry -- these .rc changes are great but seem worthy of submitting separately, since they aren't really related to the UI changes.